ipad/ViewControllers/Shift/ShiftViewController.swift

```diff
@@ -66,16 +66,32 @@ class ShiftViewController: BaseViewController {
     }
     
     override func viewWillDisappear(_ animated: Bool) {
-        print(self)
         NotificationCenter.default.removeObserver(self)
     }
     
     override func viewWillAppear(_ animated: Bool) {
+        self.updateStatuses();
         setupCollectionView()
         self.collectionView.reloadData()
         addListeners()
     }
-    
+    /// Iterates through inspections checking formDidValidate. If any form does validate, modifies all appropraite statuses to `.Errors`
```

Comment:
optimized updateStatuses so that if model is null, we don't have a crash (use of guard as found elsewhere in the file)